src/utils/scroll.ts

```diff
@@ -58,7 +58,7 @@ function getChangeScrollFunction(
     isReverse: boolean,
 ): ChangeScrollFunction {
     return (element: HTMLDivElement, scrollPosition: number, startPosition: number) => {
-        // eslint-disable-next-line no-not-accumulator-reassign/no-not-accumulator-reassign
+        // eslint-disable-next-line no-param-reassign, no-not-accumulator-reassign/no-not-accumulator-reassign
```

Comment:
We added this rule. Why we are turning it off here?

Reply:
Because of the way we call the function in setTimeout we have difficulties to return value. That is the reason we need to modify input argument